lib/ansible/modules/source_control/github_release.py

```diff
@@ -96,19 +114,26 @@ def main():
 
     repo = module.params['repo']
     user = module.params['user']
+    password = module.params['password']
     login_token = module.params['token']
     action = module.params['action']
 
     # login to github
     try:
-        gh = github3.login(token=str(login_token))
+        if user and password:
```

Comment:
Just a thought, and this was a problem I created, `user` was meant to be used of the repository that needs to be accessed, not for auth.
I think we should change `repo` to be the full path of the repository (`username/repo`) and then we can split it for use later on.